sections/main-product.liquid

```diff
@@ -693,6 +693,88 @@
                   </span>
                 </p>
               {%- endif -%}
+
+              {%- when 'complementary' -%}
+                <product-recommendations class="complementary-products quick-add-hidden no-js-hidden{% if block.settings.make_collapsible_row %} is-accordion{% endif %}" data-url="{{ routes.product_recommendations_url }}?section_id={{ section.id }}&product_id={{ product.id }}&limit={{ block.settings.product_list_limit }}&intent=complementary">
+                  {%- if recommendations.performed and recommendations.products_count > 0 -%}
+                    <aside {{ block.shopify_attributes }}{% if block.settings.make_collapsible_row %} class="product__accordion accordion"{% endif %}>
```

Comment:
I think for now it's ok as is 🤔 Since you get the heading that comes up right after. And I'm guessing this type of announcing might be expected by users that are familiar with it.